src/main/resources/lib/environmentdashboard/blockWrapper.jelly

```diff
@@ -0,0 +1,16 @@
+<?jelly escape-by-default='true'?>
```

Comment:
You also need an empty marker file in this directory called ‘taglib’

Reply:
ah!